src/app/SuccessButton.css

```diff
@@ -0,0 +1,3 @@
+.button {
+  background-color: green !important;

```

Comment:
It was the only way I could overwrite button color. Please suggest better example